lib/main_common.pm

```diff
@@ -2756,10 +2756,11 @@ sub load_ha_cluster_tests {
     # When not using a support server, node 1 setups barriers and mutex
     loadtest 'ha/barrier_init' if (get_var('HOSTNAME') =~ /node01$/ and !get_var('USE_SUPPORT_SERVER'));
 
-    # Wait for barriers to be initialized except when testing HAWK as a client
-    # or Pacemaker CTS regression tests
+    # Wait for barriers to be initialized except when testing with a client
+    # HAWK, Pacemaker CTS regression tests and CTDB
```

Comment:
Should be `or` instead of `and`, no? But it's just a very tiny issue :D

Reply:
It should be `or` :smile: